containers/centos-9.containerfile

```diff
@@ -39,3 +43,4 @@ COPY requirements.txt requirements.txt
 RUN python3 -m pip install --upgrade --ignore-installed pip \
     && python3 -m venv /venv \
     && /venv/bin/python3 -m pip install --requirement requirements.txt
+RUN pip3.12 install ovirt-engine-sdk-python
```

Comment:
Why use 2 version of pip inside the venv? If we must have python 3.12 to install ovirt-engine-sdk-python we should probably move to python 3.12.

If we don't have a better way at add a comment explaing why we do this.